dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -413,11 +476,13 @@ def test_text_data_raises_error(self):
     @mock.patch('dataprofiler.profilers.profile_builder.StructuredProfiler.'
                 '_update_row_statistics')
     @mock.patch('dataprofiler.profilers.profile_builder.StructuredColProfiler')
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'StructuredProfiler._update_correlation')
```

Comment:
if this is moved to the top, would not need to change the array value below

Reply:
i didn't change any array value below this. Which changes you mentioned specifically?

Reply:
i see, changed it now